c/include/libsbp/system_macros.h

```diff
@@ -247,7 +247,7 @@
 
 #define SBP_STATUS_REPORT_SYSTEM_STARLING (0)
 #define SBP_STATUS_REPORT_SYSTEM_PRECISION_GNSS_MODULE (1)
-#define SBP_STATUS_REPORT_SBP_MAJOR_PROTOCOL_VERSION_NUMBER_MASK (0x1ff)
+#define SBP_STATUS_REPORT_SBP_MAJOR_PROTOCOL_VERSION_NUMBER_MASK (0xff)
```

Comment:
Small error in the spec and just ran the c generator because I think it's the only other language using this feature